src/3d/qgs3dutils.h

```diff
@@ -331,6 +331,14 @@ class _3D_EXPORT Qgs3DUtils
      * \since QGIS 3.40
      */
     static QByteArray removeDefinesFromShaderCode( const QByteArray &shaderCode, const QStringList &defines );
+
+    /**
+     * Gets the maximum number of clip planes that can be used.
+     * This value depends on the OpenGL implementation. It should be at least 6.
+     *
+     * \since QGIS 3.42
+     */
+    static int getOpenGLMaxClipPlanes();
```

Comment:
How about `openGlMaxClipPlanes()`?
QGIS api avoids the `get` prefix and camel-cases acronyms too

Reply:
Done.